src/ORM/DB.php

```diff
@@ -21,6 +22,22 @@
  */
 class DB
 {
+    /**
+     * A dynamic connection that will use either a replica connection if one is
+     * available and not forced to use the 'primary' connection, or the 'primary' connection
+     */
+    public const CONN_DYNAMIC = 'dynamic';
+
+    /**
+     * The 'primary' connection name, which is the main database connection and is used for all write
+     * operations and for read operations when the 'default' connection is forced to use the 'primary' connection
```

Comment:
There is no connection named "default" anymore - I assume you meant "dynamic" here.
```suggestion
     * operations and for read operations when the 'dynamic' connection is forced to use the 'primary' connection
```

Reply:
Updated